README.md

```diff
@@ -1,13 +1,70 @@
 # Humanitec AWS Reference Architecture
 
+> TL;DR
+>
+> [Follow this tutorial to spin up your Humanitec Google Cloud Reference Architecture Implementation](README.md#how-to-spin-up-your-humanitec-aws-reference-architecture)
```

Comment:
Make it clear we are staying within the same document, like "Skip the theory? Go [here](#how-to-spin-up-your-humanitec-aws-reference-architecture) to spin up your Humanitec Google Cloud Reference Architecture Implementation".